xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-events-hibernate/xwiki-platform-legacy-events-hibernate-api/pom.xml

```diff
@@ -24,22 +24,21 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.xwiki.platform</groupId>
-        <artifactId>xwiki-platform-eventstream-stores</artifactId>
+        <artifactId>xwiki-platform-legacy-events-hibernate</artifactId>
         <version>14.5-SNAPSHOT</version>
     </parent>
-    <artifactId>xwiki-platform-eventstream-store-hibernate</artifactId>
-    <name>XWiki Platform - Event Stream - Store - Hibernate</name>
-    <description>Hibernate based events storage</description>
+    <artifactId>xwiki-platform-legacy-events-hibernate-api</artifactId>
```

Comment:
Why this name choice? why not keeping `eventstream-store-hibernate` and why adding api there?

Reply:
ok